docs/service-mesh/03-03-sidecar-proxy-injection.md

```diff
@@ -3,7 +3,7 @@ title: Sidecar Proxy Injection
 type: Details
 ---
 
-By default, the Istio sidecar injector watches all Pod creation operations on all Namespaces and injects the newly created Pods with a sidecar proxy.
+By default, the Istiod watches all Pod creation operations on all Namespaces and injects the newly created Pods with a sidecar proxy.
```

Comment:
```suggestion
By default, `istiod` watches all Pod creation operations on all Namespaces and injects the newly created Pods with a sidecar proxy.
```
I gather you mean the [Istio daemon](https://istio.io/latest/blog/2020/istiod/) here and it's not a typo, right? Just making sure.